### PR TITLE
ci: setup CircleCI with linting, testing, building & publishing

### PR DESCRIPTION
**Pull request by sasalatart**

### CODE REVIEW

File: tests/test_dummy.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# pylint: disable=C0114
```
Comment (sasalatart) [inline tests/test_dummy.py:1]: This is a dummy test file. This file should be removed as soon as we have real test files.

Comment (felipehuerta17) [reply tests/test_dummy.py:1]: Great! do you have a tutorial on the format of the tests?

**Comment by felipehuerta17** [comment]
Thanks @sasalatart for the great contrubution. I will study more about circleci.

**Comment by sasalatart** [comment]
> All good! Just included sklearn

@felipehuerta17 ok, although this PR does not use sklearn, so it should not introduce it. The first PR that uses it should introduce it. Once that PR is merged, even if it is merged before this one, then the project will be considered to have sklearn on it. This is because when I merge this PR, it will in that moment be updated to have latest code in master.

Also, remember that when you add a new dependency to `Pipfile`, you should not add it manually, but instead add it through `pipenv install <dependency>`, so that `Pipfile.lock` gets updated accordingly ;)

For these reasons, I will add a small commit removing `sklearn`

File: tests/test_dummy.py (hunk 1 above)
Comment (sasalatart) [reply tests/test_dummy.py:1]: Not at the moment, but I will look for one during the weekend :)

**Comment by sasalatart** [comment]
> Thanks @sasalatart for the great contrubution. I will study more about circleci.

If you have any questions let me know, we can have a call and I can teach you the basics

**Pull request merged**